tests/test_cookie_signing.py

```diff
@@ -62,3 +62,8 @@ def test_set_and_get_cookie_with_existing_cookie():
     REQUEST.environ['HTTP_COOKIE'] = 'cookie=true'
     REQUEST.cookie('test', 'testvalue')
     assert REQUEST.get_cookie('test') == 'testvalue'
+
+
+def test_set_and_get_cookie_http_only():
+    REQUEST.cookie('test', 'testvalue', http_only=True)
+    assert REQUEST.get_cookie('test') == 'testvalue'
```

Comment:
let's add an additional test here.

again the `http_only=True` doesn't really make sense. there should be a test like:

```python
REQUEST.cookie('test', 'testvalue', http_only=True)
assert REQUEST.get_cookie('test') == 'testvalue'

assert 'HttpOnly' in REQUEST.cookies[0][1] # [0] gets the first tuple and then the [1] gets the tuples value (where the cookie information is)
```

so just get rid of the http_only=True and it should be http_only=False and then check that the `HttpOnly` doesn't exist